src/app.js

```diff
@@ -26,52 +26,55 @@ function main(sources) {
     time: new Date()
   }]);
 
+  const intent = DOM => {
+    const changeMessage$ = sources.DOM
+      .select('.input-message')
+      .events('input')
+      .map(e => e.target.value)
+      .share();
```

Comment:
Good question!

My understanding of this is as so: each time you subscribe to an observer, it won't re-use the same values. This means you will see a value pulled through your observable for each subscriber, e.g.

``` js
const x$ = Rx.Observable.just(1).map(x => { console.log(x); return x; });
x$.subscribe();
x$.subscribe();
```

In this example will see `x` logged twice, because the value is pulled through the observable once for each subscriber.

There's probably a better way to explain this… anyhow, `share` means the value will be shared throughout all subscribers, so for this:

``` js
const x$ = Rx.Observable.just(1).map(x => { console.log(x); return x; }).share();
x$.subscribe();
x$.subscribe();
```

We will only see `x` logged once.